repos/system_upgrade/el7toel8/actors/setuptargetrepos/actor.py

```diff
@@ -80,7 +80,7 @@ def process(self):
         for used_repos in self.consume(UsedRepositories):
             for used_repo in used_repos.repositories:
                 used[used_repo.repository] = used_repo.packages
-                for repo in repo_file.data:
+                for repo in used_repo.data:
                     enabled_repos.add(repo.repoid)
```

Comment:
```suggestion
                enabled_repos.add(used_repo.repository)
```

Reply:
While this would fix the issue, I am not sure that this is actually correct - Because who says that a used repository is actually enabled?